### DISCUSSION TITLE
HA Dashboard the value 0 is displayed everywhere

### DISCUSSION THREAD

**Discussion by Oli1410** [GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200#38] [Q&A]
Thank you very much for providing this great project. 

I have now got everything up and running on the hardware side. The SVG files are also displayed cleanly in the dashboard. 

But unfortunately I am failing to display the values from PE1 Kesselim Dachboard. On froeling.local the data or values are correctly transferred but in the HA Dashboard the value 0 is displayed everywhere.

It looks as if the entities have not yet been created. Unfortunately, I don't know what else to do to get it working.

I'm still completely new to the Home Assistant.

Maybe someone can tell me briefly in keywords what to do.

Thank you very much.

Translated with DeepL.com (free version)

**Comment by GyroGearl00se** [comment]
Hi @Oli1410 ,

Can you provide some screenshots for your esp device entities within home assistant? If the reading of the values works as expected and the values are available, then the correct values should be displayed in the graphics (SVG) as well. If the issue is only related to the graphics itself and not to the entities, it might help to clear the browser cache.
